docs/guide/esql-pandas.asciidoc

```diff
@@ -0,0 +1,453 @@
+[[esql-pandas]]
+=== ES|QL and Pandas
+
+The {ref}/esql.html[Elasticsearch Query Language (ES|QL)] provides a powerful
+way to filter, transform, and analyze data stored in {es}. Designed to be easy
+to learn and use, it is a perfect fit for data scientists familiar with Pandas
+and other dataframe-based libraries as ES|QL queries produce tables with named
+columns, which is the definition of dataframes.
+
+This section shows you how to use ES|QL and Pandas together to work with
+dataframes.
```

Comment:
Two minor thoughts:
* "This page" instead of "This section" now, right?
* I'm not sure this content really does show "how to use" ES|QL and Pandas... it shows an example that the reader has to then abstract or translate to their own use cases. It's more of an illustration than a how-to. Maybe rephrase?